cves/kernel/CVE-2020-36558.yml

```diff
@@ -257,8 +251,11 @@ i18n:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    This bug is apart of the video terminal drivers subsystem. It may be a 
+    stretch to say it is related to i18n, but the subsystem had code that maps 
+    keyboard keys and special keys in the driver interface.
```

Comment:
This is false, but your note is mostly fine. You can say the subsystem may handle internationalization. But this specific vulnerability does not have to do with internationalization.

Reply:
changed